src/chat/KeywordResponses.hpp

```diff
@@ -0,0 +1,38 @@
+// chris busch
+
+#pragma once
+
+namespace chat {
+
```

Comment:
There were confusing misspellings in the old code like "wate". After poking around I realised it uses weights when calculating which response to fire back.

Reply:
cripes! :(